joint_teapot/utils/joj3.py

```diff
@@ -196,6 +196,11 @@ def generate_title_and_comment(
         + "[Joint-Teapot](https://github.com/BoYanZh/Joint-Teapot) with ❤️.\n"
     )
     for stage in stages:
+        stage_score = 0
+        for i, result in enumerate(stage["results"]):
+            stage_score += result["score"]
+        if stage_score == 0 and stage["results"][0]["comment"].strip() == "":
```

Comment:
We need to make sure every comment in this stage is empty.

`all(result["comment"].strip() == "" for result in stage["results"])`